metricbeat/module/jolokia/_meta/Dockerfile

```diff
@@ -6,7 +6,8 @@ ENV TC apache-tomcat-${TOMCAT_VERSION}
 ARG JOLOKIA_VERSION
 
 RUN apk update && \
-    apk add curl openssl ca-certificates bash
+    apk upgrade --no-cache && \
```

Comment:
It uses an old version of `curl` which fails when you build locally. So this explicit upgrade was required